docs/tutorials/qsimcirq_gcp.md

```diff
@@ -1,105 +1,122 @@
+[Google Cloud project](https://cloud.google.com/resource-manager/docs/cloud-platform-resource-hierarchy#projects).
+You can create a new one, or select a project you already created:
 
-1. Select or create a Google Cloud project.
-    - [GO TO THE PROJECT SELECTOR PAGE](https://console.cloud.google.com/projectselector2/home/dashboard)
-2. Enable billing for your project.
-    - [ENABLE BILLING](https://support.google.com/cloud/answer/6293499#enable-billing)
+1. Select or create a Google Cloud project on the
+   [project selector page](https://console.cloud.google.com/projectselector2/home/dashboard).
+2. [Enable billing](https://support.google.com/cloud/answer/6293499#enable-billing)
+   for your project.
 
-When you finish this tutorial, you can avoid continued billing by deleting the resources you created. See [Cleaning up](#cleaning-up) for more detail.
+After finishing this tutorial, you can avoid continued billing by deleting the
+resources you created. See [Clean up](#clean-up) section below for details.
 
-## Creating a GCE VM 
 
-Once you have a project enabled with a billing account, you will be able to create resources. The key resource required for this project is a Virtual Machine (VM) used to run the qsim quantum simulations. 
+## Create a GCE VM
 
+Once you have a project enabled with a billing account, you will be able to
+create resources. The key resource required for this project is a Virtual
+Machine (VM) used to run the qsim quantum simulations.
 
 ### Use the Cloud Console
 
-Connect to the Cloud Console and make sure your current project is selected. Then click on "Create" for a new VM instance:
-
-
+Connect to the Cloud Console and make sure your current project is selected.
+Then click on *Create* for a new VM instance:
 
 ![alt_text](../images/qsimcirq_gcp/image7.png )
 
-
-
 ### Build a Container Optimized VM
 
 To create the VM use the steps in sequence below:
 
-
 *   Change the name of the VM to be something meaningful like "qsim-1".
 *   Choose a Region and Zone
-    *   The values are not too important now, but for latency and availability [you may make different choice](https://cloud.google.com/compute/docs/regions-zones#available).
+    * The values are not too important now, but for latency and availability
+      [you may make different choice](https://cloud.google.com/compute/docs/regions-zones#available).
 *   Choose the machine family / series: N2
     * Other machine types, like N1 or C2, will also work
-    *   Quantum simulation can require a high level of resources, so we are selecting powerful processors, but [many choices are available.](https://cloud.google.com/blog/products/compute/choose-the-right-google-compute-engine-machine-type-for-you)
+    * Quantum simulation can require a high level of resources, so select
+      powerful processors, but [many choices are available.](https://cloud.google.com/blog/products/compute/choose-the-right-google-compute-engine-machine-type-for-you)
```

Comment:
This seems ambiguous: it should be clear here that N2 / N1 / C2 **are** powerful processors, and that the user doesn't need to shop around unless they have their own preferences.

Reply:
Reworded, PTAL